resources/META-INF/plugin.xml

```diff
@@ -79,7 +79,7 @@
         <i>Notable changes since 3.2.3</i>
 
         <ul>
-            <li>Compatibility with 2020.1</li>
+            <li>Compatibility with 2020.2 EAP</li>
```

Comment:
Make this 2020.2 - today JetBrains released that version of IntelliJ. Do the other changes also have to be considered?